src/components/PostList/PostList.tsx

```diff
@@ -1,5 +1,15 @@
 import React from 'react';
+import { Post } from '../../types/Post';
+import { PostInfo } from '../PostInfo';
 
-export const PostList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  posts: Post[];
+};
+
+export const PostList: React.FC<Props> = ({ posts }) => (
+  <div className="PostList">
```

Comment:
```suggestion
  <ul className="PostList">
```
use semantic tags for a list